pages/1.11/installing/evaluation/gcp/index.md

```diff
@@ -240,7 +272,7 @@ Terraform makes it easy to scale your cluster to add additional agents (public o
 
   ```bash
   terraform plan -out=plan.out
-  ```
+GG  ```
```

Comment:
is this intentional? looks like an accidental typo

Reply:
Thank you for the catch Jan! That was unintentional.